.github/workflows/publish-helm-chart.yml

```diff
@@ -0,0 +1,40 @@
+# Publish the mastdodon helm chart to mastodon.github.io/mastodon as a .tgz file.
+# helm/chart-releaser-action will push an index.yaml file to the gh-pages branch with a reference to the .tgz file.
+# The tgz file is stored as a release asset in github.
+name: Publish helm chart
+
+concurrency: release-helm
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'main'
```

Comment:
I think it's worth thinking carefully about and documenting how this will work in practice for the person merging helm PRs, especially given that this will run on every commit and will create a chart release as soon as possible

Reply:
Yeah that makes sense, I'll make a proposal based on what we do at my company for helm chart changes/versioning

Reply:
> especially given that this will run on every commit and will create a chart release as soon as possible

It will create chart release only when we explicitly bump the chart version, right ?

Reply:
@mtparet correct